### DISCUSSION TITLE
The sel option in the input.json

### DISCUSSION THREAD

**Discussion by andy90** [deepmodeling/deepmd-kit#1064] [Q&A] [answered]
Hi, I have a question about the sel option in the input.json. According to the document, sel[i] defines the number of neighbour atoms inside the cutoff radius:

`sel gives the maximum possible number of neighbors in the cut-off radius. It is a list, the length of which is the same as the number of atom types in the system, and sel[i] denote the maximum possible number of neighbors with type i.`

But I found this documentation a bit confusing. Let's take water for example. Let's assume that, around the Oxygen atom, there are 100 Oxygen and 200 Hydrogen within the neighbor. But around the Hydrogen atom, there are 200 Oxygen and 400 Hydrogen within the neighbor. In that case, how should we define sel[i]?

**Comment by J-You** [comment]
in my mind, it only means coordination number without consideration of the type of neighbors.

**Comment by dmh1998dmh** [comment, marked as answer]
In my word, sel[i] is the maximum number of type i atoms in a sphere (r=rcut), whose center can be any atom in your system.
